src/osdc/Objecter.h

```diff
@@ -1233,7 +1233,7 @@ class Objecter : public md_config_obs_t, public Dispatcher {
   using unique_lock = std::unique_lock<decltype(rwlock)>;
   using shared_lock = boost::shared_lock<decltype(rwlock)>;
   using shunique_lock = ceph::shunique_lock<decltype(rwlock)>;
-  ceph::timer<ceph::mono_clock> timer;
+  ceph::timer<ceph::coarse_mono_clock> timer;
 
```

Comment:
This probably won't have an effect, since the libstdc++ condition variable always uses the same clock whatever you pass it. Doesn't hurt anything, though.